lib/api/_loaders/_base-loader.js

```diff
@@ -301,29 +301,28 @@ class BaseLoader extends EventEmitter {
     return args;
   }
 
-  isRestrictedNamespace(parent, namespace) {
-    if (!parent) {
-      return false;
```

Comment:
This feature to restrict certain namespaces from being loaded onto page-objects (or to load only certain namespaces onto page-objects) was added separately in page objects, so this is no longer required.